src/waterfall/polarplot.cpp

```diff
@@ -114,7 +114,7 @@ void PolarPlot::draw(const QVector<double>& points, float angle, float initPoint
 
 void PolarPlot::updateMouseColumnData()
 {
-    static const float rad2grad = 200/M_PI;
+    static const float rad2grad = 200.0f/M_PI;
```

Comment:
if you are using M_PI (that's a define, and not a proper value) perhaps you wan't to use boost for that:
boost::math::constants::pi<float>();

Reply:
yep, but adding a single (and big) dependency just because of M_PI is not a good approach.